plugins/inputs/t128_graphql/config.go

```diff
@@ -0,0 +1,76 @@
+package t128_graphql
+
+import (
+	"strings"
+)
+
+//Config stores paths and paths to fields, tags and predicates used by queryBuilder and responseProcessor
+type Config struct {
+	QueryPath  string
+	Predicates map[string]string
+	Fields     map[string]string
+	Tags       map[string]string
+	RawFields  map[string]string //Fields is used with RawFields to build the query
+	RawTags    map[string]string
+}
+
+//LoadConfig converts an entry point into a corresponding responsePath, queryPath and predicates
+func LoadConfig(entryPoint string, fieldsIn map[string]string, tagsIn map[string]string) *Config {
+	config := newConfig()
+	responsePath := "/data/"
+	queryPath := ""
+	predicates := map[string]string{}
+
+	pathElements := strings.Split(entryPoint, "/")
+	for idx, element := range pathElements {
+		parenIdx := strings.Index(element, "(")
```

Comment:
paren -> parent?

Reply:
Confused myself too. This should be `parenthesisIdx`